.github/workflows/python.yml

```diff
@@ -91,7 +91,7 @@ jobs:
       - name: Install Python
         uses: actions/setup-python@v2
         with:
-          python-version: 3.9
+          python-version: "3.10"
```

Comment:
not sure we want to keep running tests on 3.10 only, but it seems like the matrix (with multiple pythoon version) steps are only used for builds tests and not actual tests

Reply:
No for tests, 1 version is usually enough. (There is nothing very new to python world in this lib)